src/simulator/services/resources/directory.py

```diff
@@ -63,7 +63,7 @@ def load(self, name: str, load_function: Callable[['Directory', str], Any] = Non
         if not load_function:
             load_function = Directory._default_load
         obj: Any = load_function(self, name)
-        if obj:
+        if obj is not None:
```

Comment:
Revert this; we need it to fail the load if the object is, say an empty map because the generation fucked up somewhere. This if statement will print Loaded in the case of an empty map object (or something like that, where the file exists but it has like nothing in it).

Reply:
fixed